zwave_js_server/__main__.py

```diff
@@ -62,9 +62,9 @@ def log_value_updated(event: dict) -> None:
         value = event["value"]
 
         if node.device_config:
-            description = node.device_config["description"]
+            description = node.device_config.description
```

Comment:
This may return `None`. Maybe that's ok.

Reply:
yeah, no problem